ci-operator/step-registry/ipi/conf/azure/provisioned-des/ipi-conf-azure-provisioned-des-ref.yaml

```diff
@@ -9,6 +9,10 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
-  credentials:
+  env:
+  - name: ENABLE_DES_DEFAULT_MACHINE
+    default: "false"
+    documentation: |-
+      Adds provisioned disk ecncryption set section under DefaultMachinePlatform in the install-config.yaml
```

Comment:
a typo in the documentation 
"disk ecncryption set" -> "disk encryption set"

Reply:
maybe we can update the documentation a bit more clearly like:
```
Adds provisioned disk encryption set section under DefaultMachinePlatform in the install-config.yaml:
* "false" (default) - Adds provisioned disk encryption set section under compute/controlPlane 
* "true" - Adds provisioned disk encryption set section under DefaultMachinePlatform, which also encrypts the default storage class from 4.13
```

Reply:
updated